src/explore.ts

```diff
@@ -0,0 +1,296 @@
+import * as vscode from 'vscode';
+import * as fs from 'fs';
+import * as dump from './objdump/dumper';
+
+const objDumpPathKey = 'codeart-binexplore.objdumpPath';
+const objDumpOptionsKey = 'codeart-binexplore.objdumpOptions';
+
+const defaultObjDumpPath = '/usr/bin/objdump';
+const defaultObjDumpOptions = '-d -S';
+
+let outputChannel: vscode.OutputChannel;
+let statusBar: vscode.StatusBarItem;
+
+/**
+ * This method is called when the extension is activated.
+ * @param context The context in which the extension is activated.
+ */
+export async function activate(
+  context: vscode.ExtensionContext,
+  extensionOutputChannel: vscode.OutputChannel
+) {
+  const disposable = vscode.workspace.onDidChangeConfiguration(
+    await handleConfigurationChange
```

Comment:
"'await' has no effect on the type of this expression.ts(80007)"
```suggestion
    handleConfigurationChange
```

Reply:
Done